packages/authentication/azure/package.json

```diff
@@ -32,7 +32,7 @@
 	"homepage": "https://github.com/microsoft/kiota-typescript#readme",
 	"dependencies": {
 		"@azure/core-auth": "^1.5.0",
-		"@microsoft/kiota-abstractions": "*",
+		"@microsoft/kiota-abstractions": "file:../../abstractions/",
```

Comment:
Out of curiosity, will the published package be affected by this? Should it point to a version range? Or this won't be breaking as it may look for file/folder?